cal/cal/models.py

```diff
@@ -584,6 +584,40 @@ def query(self, calendar_ids=None, start=None, end=None):
         return events_qs
 
 
+    def queryWeek(self, calendar_ids=None, start=None, end=None):
+        if calendar_ids and self.calendar:
+            message = "Calendar ids can only be specified for ColorCategories without a calendar field attached."
```

Comment:
calendar_id refers to the unique calendar id provided by google, and is a string. It is also the primary key of the GCalendar model. Wheneve it's just `calendar`, it's referring to the `GCalendar` objects - you can infer this from how `calendar` is set in the code.